### ISSUE TITLE
menuitem "Jump to original location" has no id

### ISSUE THREAD

**Issue by magicp-github** [devtools-html/debugger.html#4607]
**Steps to reproduce:**
Inspect menuitem "Jump to original location"

**Actual Results:**
menuitem "Jump to original location" has no id. It is difficult to identify a menuitem in other locales.
![jump-to-original-location](https://user-images.githubusercontent.com/13914314/32468792-12e76e3e-c393-11e7-9e5b-4cc1e74c4120.png)

**Expected Results:**
It should have id.
